monash-human-power/MHP_DAS
# Add Sample Micropython

## Description

A sample MQTT test that runs on ESP32 using micro-python. 


## Screenshots

![20200518_122621_1](https://user-images.githubusercontent.com/23159604/83381871-2897be80-a425-11ea-888b-852d88a20e0e.gif)


## Steps to Test

In order to run this code you need an ESP32:
1. Use a tool such as ampy or rshell to copy all of the files to the ESP32
2. Restart the ESP32 and you should see an output when using mosquitto_sub or the DAS data logger script 

If you want to turn on and off the LED remotely send data to the v3/wireless-module/{MODULE_ID}/LED topic with either  "on"  or "off"
